### PR TITLE
feat: [PIE-4904]: pipeline list view minor UX changes after review 

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

- pipeline list view minor UX changes after review
- correct some existing icons, text, css etc

#### Screenshots

image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by udayvunnam** [comment]
retrigger titlecheck

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 6 Issues 
* Code issue 6 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10638)

File: src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx [hunk 1]
```diff
@@ -250,18 +254,17 @@ export const MenuCell: CellType = ({ row, column }) => {
...
-          className={css.actionButton}
-          icon="more"
+          icon="Options"
           onClick={e => {
             e.stopPropagation()
             setMenuOpen(true)
           }}
         />
-        <Menu style={{ minWidth: 'unset' }} onClick={e => e.stopPropagation()}>
+        <Menu style={{ minWidth: 'unset', backgroundColor: 'unset' }} onClick={e => e.stopPropagation()}>
```
Comment (udayvunnam) [inline src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx:267]: The default BP menu uses an odd color unset it so popover background is applied

File: src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx [hunk 2]
```diff
@@ -250,18 +254,17 @@ export const MenuCell: CellType = ({ row, column }) => {
...
         }}
         className={Classes.DARK}
-        position={Position.BOTTOM_RIGHT}
+        position={Position.LEFT}
       >
         <Button
           minimal
-          className={css.actionButton}
-          icon="more"
+          icon="Options"
```
Comment (udayvunnam) [inline src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx:261]: use proper icon so rotation is not required.

**Review by kapilkumawat86 — APPROVED** [review]

File: src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx (hunk 1 above)
Comment (dhruvUpadhyay07) [reply src/modules/70-pipeline/pages/pipeline-list/PipelineListTable/PipelineListCells.tsx:267]: we can remove these propagation functions now

**Review by dhruvUpadhyay07 — APPROVED** [review]

**Pull request merged**